explore/src/main/scala/explore/Routing.scala

```diff
@@ -116,7 +116,10 @@ object Routing {
             targetTab
           )
           | staticRoute("/configurations", ConfigurationsPage) ~> render(SequenceEditor())
-          | staticRoute("/constraints", ConstraintsBasePage) ~> renderP(constraintSetTab))
+          | staticRoute("/constraints", ConstraintsBasePage) ~> renderP(constraintSetTab)
+          | dynamicRouteCT(
+            ("/constraints/obs" / id[Observation.Id]).xmapL(ConstraintsObsPage.obsId)
```

Comment:
Will there be a url for a given obs in constraints? Isn't that kind of the inverse of the hirerachy? as in obs have constraints?

Reply:
Well, isn't editing the constraint for an observation in the constraints tab kind of the inverse of the hierarchy? 😆 Anyway, it seems consistent to me, but I can remove it if desired.

Reply:
Right, it matches what we see on screen. I was thinking it didn't make sense to do that but looking at it in practice it doesn't seem too bad